doc/classes/AudioEffectLowPassFilter.xml

```diff
@@ -1,8 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="AudioEffectLowPassFilter" inherits="AudioEffectFilter" category="Core" version="3.0.alpha.custom_build">
 	<brief_description>
+		Adds a Low Pass filter to the Audio Bus.
 	</brief_description>
 	<description>
+		The effect cuts frequences higher than the [code]Cutoff HZ[/code], and lets lower frequences pass.
```

Comment:
- Remove the comma before "and"
- "Cuts frequencies above the [member cutoff_hz]"